src/components/Playlists.tsx

```diff
@@ -40,21 +41,19 @@ class Playlists extends Component<PlaylistsProps, PlaylistsState> {
       <PlaylistFilter placeholder="Start typing to filter for your playlist" onChange={ this.filterPlaylist } value={ this.state.playlistFilter }/>
       { this.props.playlists.length > 0 ?
         <PlaylistContainer>
-          <ul>
             <PlaylistSelectForm>
             { this.props.playlists.filter((playlists) => this.checkIncludes(playlists.name.toLowerCase(), this.state.playlistFilter.toLowerCase())).map((pl) => (
               <PlaylistDiv key={pl.uri}>
                 <PlaylistImage src={pl.images[0]?.url} alt="Playlist art"></PlaylistImage>
                 <PlaylistItem>
                   <PlaylistName>{pl.name}</PlaylistName>
                   <PlayButton onClick={() => this.startPlayback(pl.uri)} type="button">
-                    <img style={{width: "50px", height: "50px", margin: 0}} src={playIcon}></img>
+                    <PlayButtonIcon src={playIcon}></PlayButtonIcon>
```

Comment:
Created a new styled component for the play button icon so that applying a mobile break point is a little bit easier than doing it in-line